core-web/libs/sdk/client/README.md

```diff
@@ -127,11 +127,17 @@ const pageData = await client.page.get('/your-page-path', {
 });
 ```
 
+
+### Warning
+
+If you are updating from a version that is lower than `0.0.1-beta.29`, be aware that the response from the `client.page.get` method changed the access to the page value from `page` to `pageAsset`.
+This change was made to avoid redundancy on access inside of `page` object Ex. `page.page.title` -> `pageAsset.page.title`.
+
```

Comment:
Add a before and after example, like:

**Before**
```ts
const { page, content } = await client.page.get()
```

**Now**
```ts
const { pageAsset, content } = await client.page.get()
```

Reply:
Added!